binding.cc

```diff
@@ -542,9 +542,9 @@ NAN_METHOD(crypto_generichash_instance) {
   unsigned long long output_length = crypto_generichash_bytes();
 
   if (info[1]->IsObject()) {
-    output_length = CLENGTH(info[1]->ToObject());
+    output_length = CLENGTH(info[1]->ToObject(v8::Isolate::GetCurrent()->GetCurrentContext()).ToLocalChecked());
   } else if (info[1]->IsNumber()) {
-    output_length = info[1]->Uint32Value();
+    output_length = info[1].As<v8::Uint32>()->Value();
```

Comment:
Something similar could be said here: `output_length = Nan::To<uint32_t>(info[1]).FromJust();` (perhaps with a `IsJust()` check. Curious about the differences if you know?

Reply:
Since we're checking `IsNumber`, we can do `output_length = info[1].As<v8::Number>()->Value();` here too. Otherwise this will have to change to checking for integer.

Reply:
Btw, the Nan docs seem to do `double value = info[0]->IsNumber() ? Nan::To<double>(info[0]).FromJust() : 0;`
I guess it's a matter of taste whether you need to go through a `Maybe`.

Reply:
But if we want to preserve previous behaviour it should be a uint32_t no?